### ISSUE TITLE
[Features] Better Icons

### ISSUE THREAD

**Issue by JingJang** [zixaphir/Stable-Diffusion-Webui-Civitai-Helper#28]
For icons:

1) When you select rename, the default value is an empty string. Consider filling it with current name.
2) Maybe consider using a link for "Open this model civitai" rather than an click event
3) Gray out icons when there is no data associated with it. So if no url link or prompt information then appropriate icon should be grayed out.

**Comment by zixaphir** [comment]
>When you select rename, the default value is an empty string. Consider filling it with current name.

This is a good idea. I'll get that in soon.

>Maybe consider using a link for "Open this model civitai" rather than an click event

This is largely implemented the way it is because of the the "Open model Url on the user's client side" option. I can definitely put in just a link if that's on, which it is by default. Honestly, I don't know why you'd disable that option, but I'm sure somebody does so I can't just remove it.

>Gray out icons when there is no data associated with it. So if no url link or prompt information then appropriate icon should be grayed out.

I think I can do this, yeah.

**Comment by JingJang** [comment]
Can you also consider brining back the add "trigger words" button. There are checkpoints that have triggers and sometimes it does not seem to add the trigger when you click the a lora. Not sure why. I know the lora has trigger words but they aren't added.

**Comment by zixaphir** [comment]
>Can you also consider brining back the add "trigger words" button. There are checkpoints that have triggers and sometimes it does not seem to add the trigger when you click the a lora. Not sure why. I know the lora has trigger words but they aren't added.

You can re-enable it or show/hide other buttons in the settings menu.

>In the rename box, could you please include the title of the actual model from the script. Sometimes models are just named numbers and Civitai title would give an idea what you should call it.

I'm thinking about options renaming models by default that would be better for this.

**Comment by JingJang** [comment]
I should have checked settings. Sorry.

> I'm thinking about options for renaming models by default that would be better for this.

No don't do that. Most of the titles are long winded with lots of unnecessary descriptive terms.

**Comment by zixaphir** [comment]
> I should have checked settings. Sorry.
> 
> > I'm thinking about options for renaming models by default that would be better for this.
> 
> No don't do that. Most of the titles are long winded with lots of unnecessary descriptive terms.

Of course. This would be optional, and off by default. I'm sorry my wording was bad.
